src/Shared/Microsoft.DotNet.Scaffolding.Shared/Project/ProjectModifierHelper.cs

```diff
@@ -379,25 +380,41 @@ internal static bool FilterOptions(string[] options, CodeChangeOptions codeChang
 
         internal static async Task<Document> AddTextToDocument(Document fileDoc, IEnumerable<CodeSnippet> codeChanges)
```

Comment:
Add a small comment on how this method is working. As far as I can tell, it's adding blocks in sequential order found in the code modifier configs??